build-test-deploy.yml

```diff
@@ -153,11 +153,11 @@ steps:
   displayName: 'Publishing assets to Github'
 
 - task: NuGetCommand@2
- inputs:
-   command: 'push'
-   packagesToPush: '$(Build.ArtifactStagingDirectory)/**/*.nupkg;!$(Build.ArtifactStagingDirectory)/**/*.symbols.nupkg'
-   nuGetFeedType: 'internal'
-   publishVstsFeed: '5ac05815-e952-41a1-9899-54474b67ab5d'
- continueOnError: true # Allow failure - if the nuget package version isn't updated, we don't want to push a new package, so it's ok to fail 
- condition: and(succeeded(), startsWith(variables['Build.SourceBranch'], 'refs/tags'))
- displayName: Push to NuGet Repository
+  inputs:
+    command: 'push'
+    packagesToPush: '$(Build.ArtifactStagingDirectory)/**/*.nupkg;!$(Build.ArtifactStagingDirectory)/**/*.symbols.nupkg'
+    nuGetFeedType: 'internal'
+    publishVstsFeed: '5ac05815-e952-41a1-9899-54474b67ab5d'
+  continueOnError: true
+  condition: and(succeeded(), startsWith(variables['Build.SourceBranch'], 'refs/tags'))
+  displayName: Push to NuGet Repository
```

Comment:
@peterjgrainger the idea is that this would be a separate github workflow.. we only want to do a pre-release on azure and trigger the actual release from github.. 
so i guess that last task should move to a github workflow?!